docs/solutions/connect_vcns_using_multiple_vnics/vcn1.tf

```diff
@@ -0,0 +1,195 @@
+variable "tenancy_ocid" {}
+variable "user_ocid" {}
+variable "fingerprint" {}
+variable "private_key_path" {}
+variable "compartment_ocid" {}
+variable "region" {}
+variable "ssh_public_key" {}
+variable "ssh_private_key" {}
+
+# Choose an Availability Domain
+variable "AD" {
+    default = "1"
+}
+
+variable "InstanceShape" {
+    default = "VM.Standard1.2"
+}
+
+variable "InstanceShape2" {
+    default = "VM.Standard1.1"
+}
+
+variable "InstanceImageOCID" {
+    type = "map"
+    default = {
+        // See https://docs.us-phoenix-1.oraclecloud.com/images/
+        // Oracle-provided image "Oracle-Linux-7.4-2018.02.21-1"
+        us-phoenix-1 = "ocid1.image.oc1.phx.aaaaaaaaupbfz5f5hdvejulmalhyb6goieolullgkpumorbvxlwkaowglslq"
+        us-ashburn-1 = "ocid1.image.oc1.iad.aaaaaaaajlw3xfie2t5t52uegyhiq2npx7bqyu4uvi2zyu3w3mqayc2bxmaa"
+        eu-frankfurt-1 = "ocid1.image.oc1.eu-frankfurt-1.aaaaaaaa7d3fsb6272srnftyi4dphdgfjf6gurxqhmv6ileds7ba3m2gltxq"
+        uk-london-1 = "ocid1.image.oc1.uk-london-1.aaaaaaaaa6h6gj6v4n56mqrbgnosskq63blyv2752g36zerymy63cfkojiiq"
+    }
+}
+
+# The First VCN
+variable "vcn_cidr" {
+    default = "10.0.0.0/16"
+}
+
+variable "mgmt_subnet_cidr" {
+    default = "10.0.0.0/24"
+}
+
+variable "private_subnet_cidr" {
+    default = "10.0.1.0/24"
+}
+
+provider "oci" {
+    tenancy_ocid = "${var.tenancy_ocid}"
+    user_ocid = "${var.user_ocid}"
+    fingerprint = "${var.fingerprint}"
+    private_key_path = "${var.private_key_path}"
+    region = "${var.region}"
+}
+
+data "oci_identity_availability_domains" "ADs" {
+    compartment_id = "${var.tenancy_ocid}"
+}
+
+resource "oci_core_virtual_network" "CoreVCN" {
+    cidr_block = "${var.vcn_cidr}"
+    compartment_id = "${var.compartment_ocid}"
+    display_name = "VCN-1"
+}
+
+# First VCN configuration
+resource "oci_core_internet_gateway" "MgmtIG" {
+    compartment_id = "${var.compartment_ocid}"
+    display_name = "MgmtIG"
+    vcn_id = "${oci_core_virtual_network.CoreVCN.id}"
+}
+
+resource "oci_core_route_table" "MgmtRouteTable" {
+    compartment_id = "${var.compartment_ocid}"
+    vcn_id = "${oci_core_virtual_network.CoreVCN.id}"
+    display_name = "MgmtRouteTable"
+    route_rules {
+        cidr_block = "0.0.0.0/0"
+        network_entity_id = "${oci_core_internet_gateway.MgmtIG.id}"
+    }
+}
+
+resource "oci_core_security_list" "MgmtSecurityList" {
+    compartment_id = "${var.compartment_ocid}"
+    display_name = "MgmtSecurityList"
+    vcn_id = "${oci_core_virtual_network.CoreVCN.id}"
+
+    egress_security_rules = [{
+        protocol = "all"
+        destination = "0.0.0.0/0"
+    }]
+
+    ingress_security_rules = [{
+        protocol = "all"
+        source = "${var.vcn_cidr}"
+    },
```

Comment:
nitpick: formatting is a little strange... can you run "terraform fmt" in this directory?